src/functions/tarsUpload/domain/upload/TARSPayloadConverter.ts

```diff
@@ -1,6 +1,6 @@
 import { ITARSPayloadConverter } from './ITARSPayloadConverter';
 import { TestResultSchemasUnion } from '@dvsa/mes-test-schema/categories';
-import { PassCompletion } from '@dvsa/mes-test-schema/categories/Common';
+import { PassCompletion } from '@dvsa/mes-test-schema/categories/common';
```

Comment:
I think the import paths are not case sensitive. Anyway lowercase `common` is the correct way.